Twig/FrameworkExtension.php

```diff
@@ -86,4 +90,15 @@ public function convertToTranslation($stringToConvert)
 
         return trim($stringToConvert, '.');
     }
+
+    /**
+     * Capitalize the first letter in a word/sentence
+     *
+     * @param string $word
+     * @return bool
+     */
+    public function ucfirst($word)
```

Comment:
Zouden we niet gewoon typehints gebruiken?

Reply:
fixed 9bc0438e041739b303a7b2255a564f7c7fdba5f6